vignettes/email-overview.Rmd

```diff
@@ -0,0 +1,141 @@
+---
+title: "Überblick Open-Access-Metadaten-Schnelltest"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Überblick Open-Access-Metadaten-Schnelltest}
```

Comment:
muggle/pkgdown has a more succinct version: https://subugoe.github.io/muggle/reference/build_site2.html

we shouldn't really build *vignettes*, just articles, makes things much easier.